timeseries/tests/smoketests/test_all_models.py

```diff
@@ -88,6 +88,7 @@ def generate_train_and_test_data(
     "SimpleFeedForward": DUMMY_MODEL_HPARAMS,
     "TemporalFusionTransformer": DUMMY_MODEL_HPARAMS,
     "TiDE": DUMMY_MODEL_HPARAMS,
+    "Toto": {"num_samples": 20, "device": "cpu"},
```

Comment:
Maybe we can keep this even lower for better efficiency? Like `"num_samples": 5"` even